dashboard/consumption/management/commands/import.py

```diff
@@ -1,8 +1,65 @@
+import logging
+import time
+
+import pandas as pd
 from django.core.management.base import BaseCommand
+from django.db.models import Sum
+from django.db.models.functions import TruncDay
+
+from consumption.models import CalculatedConsumption, Consumption, User
+from dashboard.settings import APP_LOGGER_NAME
+
+logger = logging.getLogger(f'{APP_LOGGER_NAME}.{__name__}')
```

Comment:
djangoは1つのプロジェクト内に複数のアプリケーションを内包することが一般的に多いと思います。その場合`APP_LOGGER_NAME`は少々使いやすさに欠けるので、下記のような書き方で十分かと思います。
https://docs.djangoproject.com/en/4.1/howto/logging/#make-a-basic-logging-call
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
よりAdvancedな改善としては、全てのソースコードにこの設定を書くのは冗長として、Mixin等で共通化するなどがあるかなと思います。
```python
class LoggingMixin:
    @property
    def logger(self) -> logging.Logger:
        if not hasattr(self, "_logger"):
            self._logger = logging.getLogger(
                self.__class__.__module__ + "." + self.__class__.__name__
            )
        return self._logger

class Command(BaseCommand, LoggingMixin):
    def handle(self, *args, **kwargs):
        self.logger.info("Log something")
```

Reply:
今まで共通化せず使っていたので大変勉強になりました🙇‍♀️
教えてくださりありがとうございます！